tests/aws/aio/aiomoto_fixtures.py

```diff
@@ -0,0 +1,184 @@
+"""
+AWS asyncio test fixtures
+"""
+
+import aiobotocore.client
+import aiobotocore.config
+import pytest
+
+from tests.aws.aio.aiomoto_services import MotoService
+from tests.aws.utils import AWS_ACCESS_KEY_ID
+from tests.aws.utils import AWS_SECRET_ACCESS_KEY
+
+
+#
+# Asyncio AWS Services
+#
+
+
+@pytest.fixture
+async def aio_aws_batch_server():
+    async with MotoService("batch") as svc:
+        svc.reset()
+        yield svc.endpoint_url
+
+
+@pytest.fixture
+async def aio_aws_cloudformation_server():
+    async with MotoService("cloudformation") as svc:
```

Comment:
I think we can replace this line across tests with a parameterized fixture

Reply:
see https://stackoverflow.com/questions/18011902/pass-a-parameter-to-a-fixture-function/60148972#60148972  I think

Reply:
then this whole section can be collapsed

Reply:
Explored some ideas along these lines but failed so far, will try to follow up with these tips (or feel free to take it and run with it on a patch on this patch, but this PR is WIP and this branch might pull the rug until it's merged, if merged).

Reply:
really appreciate all the help!  Just throwing ideas out there, some wrong :)

Reply:
See `test_aio_aws_client` and the fixture it uses, which returns a client generator.  The `test_aio_aws_client` could be easily parameterized to run all available clients if it has some generic assertions.